backend/users/migrations/0008_alter_bearertoken_expires_at.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.3 on 2024-08-06 16:29
+
+import django.utils.timezone
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("users", "0007_implicitbearertoken_id_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="bearertoken",
+            name="expires_at",
+            field=models.DateTimeField(default=django.utils.timezone.now),
```

Comment:
By default, all existing tokens will expire immediately at the time of the migration. Is that what you want?

Reply:
I think the idea was rather to set an immediate expiration date for token that does not have one

Reply:
Updated, thank you